eng/scripts/Test-Code.ps1

```diff
@@ -73,67 +97,79 @@ function CopyNativeBinaryToTestDirs {
     param(
         [Parameter(Mandatory=$true)]
         [string]$nativeBinaryPath,
-        [string[]]$areas = @()
+        [string[]]$testsRootDirs
     )
     Write-Host "Copying native AzureMcp to test directories"
 
-    if (!$areas) {
-        $testDirectories = Get-ChildItem -Path $RepoRoot -Recurse -Filter "*.LiveTests" -Directory
-    } else {
-        $testDirectories = @()
-        foreach ($area in $areas) {
-            $areaName = $area.ToLower()
-            $areaPath = $areaName -eq 'core' ? "$RepoRoot/core/tests" : "$RepoRoot/areas/$areaName/tests"
-            if (Test-Path $areaPath) {
-                $areaTestDirectories = Get-ChildItem -Path $areaPath -Recurse -Filter "*.LiveTests" -Directory
-                $testDirectories += $areaTestDirectories
-            }
-        }
+    $testProjectDirs = @()
+    foreach ($testsRootDir in $testsRootDirs) {
+        $areaTestProjectDirs = Get-ChildItem -Path $testsRootDir -Recurse -Filter "*.LiveTests" -Directory
+        $testProjectDirs += $areaTestProjectDirs
     }
 
-    foreach ($testDir in $testDirectories) {
+    foreach ($testDir in $testProjectDirs) {
```

Comment:
Given `$testProjectDirs` is only used here, why separate out the logic for handling the test dirs here? Can't this logic be merged into the other loop and then we can just iterate over the results of `Get-ChildItem`.